dio/pubspec.yaml

```diff
@@ -1,14 +1,20 @@
 name: dio
+version: 5.3.0
 
 description: |
   A powerful HTTP networking package,
   supports Interceptors,
   Aborting and canceling a request,
   Custom adapters, Transformers, etc.
+topics:
+  - dio
+  - http
+  - network
+  - interceptor
```

Comment:
```suggestion
  - interceptor
  - middleware
```

Reply:
Well we can only have 5 topics for the package.

Reply:
Should I remove Web then?

Reply:
I see, good question. Middleware is, as far as I know, the more commonly used term for what dio calles interceptor, so that's why I suggested that. I don't have any particular strong opinion on it, so the current selection is also fine for me.